test/utils.test.js

```diff
@@ -0,0 +1,63 @@
+"use strict";
+
+const utils = require("../utils");
+
+global.fetch = jest.fn(() =>
+    Promise.resolve({
```

Comment:
Are the four-spaces indentation here expected? I think the usual codestyle is 2-spaces, but prettier might not run on `.test.js` files.

Reply:
Oh right, looks like prettier didn't format this file